README.md

```diff
@@ -184,6 +184,23 @@ Notes:
   * `"disk"`: Boot from local hard disk.
   * `"none"`: Reset boot order to default.
 
+### Checking a node's power status
+
+`GET /node/{node_id}/power_status`
+
+Response body:
+
+```json
+{
+    "power_status": "<IPMI chassis power status response>"
+}
+```
+
+Notes:
+
+* Returns a JSON string that describes the node's power state.
+* Response examples: "Chassis Power is on" or "Chassis Power is off"
```

Comment:
It's a minor thing, but since this is intended to be used programmatically, it feels like it would be more natural to just return the strings `"on"` and `"off"`, rather than passing through the output from ipmitool unchanged.

Reply:
sure, that sounds good to me